client/src/apis/index.ts

```diff
@@ -1,7 +1,7 @@
 import axios from 'axios';
 
 const axiosInstance = axios.create({
-  baseURL: process.env.NEXT_PUBLIC_API_BASE_URL
```

Comment:
revert this changes
we will use `baseURL: process.env.NEXT_PUBLIC_API_BASE_URL`